components/Converters/samples/Converters.md

```diff
@@ -12,151 +12,71 @@ issue-id: 0
-
-```xaml
-<Page.Resources>
-    <converters:DoubleToVisibilityConverter x:Key="GreaterThanToleranceVisibilityConverter" GreaterThan="65.0"/>
-</Page.Resources>
-```
-
-and use it like this :
-
-```xaml
-<Button x:Name="ScrollBackButton"
-        Visibility="{Binding ScrollableWidth, Converter={StaticResource GreaterThanToleranceVisibilityConverter}, ElementName=ScrollViewer, FallbackValue=Collapsed, TargetNullValue=Collapsed}"/>
-```
-
-## EmptyObjectToObjectConverter Examples
-
-`EmptyObjectToObjectConverter`, `EmptyCollectionToObjectConverter`, and `EmptyStringToObjectConverter` work similarly to the `BoolToObjectConverter` except using `EmptyValue` and `NotEmptyValue` instead of `TrueValue`/`FalseValue`.
-
-They inspect the type of 'empty'/null object value and return the specific value `EmptyValue` or `NotEmptyValue` based on the result.
-That result can also be inverted withe a ConverterParameter.
-
-For instance you can generalize the `CollectionVisibilityConverter` using the `EmptyCollectionToObjectConverter`:
+## StringFormatConverter
+This allows you to format a string property upon binding wrapping [string.Format](/dotnet/api/system.string.format?view=netstandard-2.0).  
+It only allows for a single input value (the binding string), but can be formatted with the regular string.Format
+methods.
 
-```xaml
-<Page.Resources>
-    <converters:EmptyCollectionToObjectConverter x:Key="CollectionVisibilityConverter" EmptyValue="Collapsed" NotEmptyValue="Visible"/>
-</Page.Resources>
-```
+> [!Sample StringFormatConverterSample]
 
-this can be used as follows to hide a list with no items and instead show text through inversion with the ConverterParameter:
+## StringVisibilityConverter
+Converts a string value into a Visibility value (if the value is null or empty returns a collapsed value).
 
-```xaml
-<ListView Visibility="{Binding Path=MyCollectionValue, Converter={StaticResource CollectionVisibilityConverter}}" />
+> [!Sample StringVisibilityConverterSample]
 
-<TextBlock Text="No Items." Visibility="{Binding Path=MyCollectionValue, Converter={StaticResource CollectionVisibilityConverter}, ConverterParameter=True}">
-```
+## TaskResultConverter
+Converter that can be used to safely retrieve results from `Task<TResult>` instances. See MVVM Toolkit Gallery for an example.
 
-## StringFormatConverter Examples
+## TypeToObjectConverter
+Returns an object or another, depending on whether the type of the provided value matches another provided Type.
+> [!Sample TypeToObjectConverterSample]
 
-`StringFormatConverter` allows you to format a string property upon binding wrapping [string.Format](/dotnet/api/system.string.format?view=netstandard-2.0).  
-It only allows for a single input value (the binding string), but can be formatted with the regular string.Format
-methods.  First, add it to your page resources:
-
-> [!Sample StringFormatConverterSample]
+## VisibilityToBoolConverter
+Converts a `Visibility` enumaration to a boolean value.
```

Comment:
Maybe for this one we just have a toggleswitch which toggles showing something, and then a textbox which binds to the visibility of that and outputs true/false to a TextBlock?